### PR TITLE
handle aiohttp and asyncio exceptions

### PR DESCRIPTION
**Pull request by kubealex**
Introduces:
- aiohttp.ClientError hierarchy for getProblems and updateProblems
- asyncio.TimoutError and asyncio.CancelledError to handle timeouts on the aiohttp.get requests.

### CODE REVIEW

File: extensions/eda/plugins/event_source/dt_esa_api.py [hunk 1]
```diff
@@ -23,20 +23,22 @@ async def getproblems(dt_host: str, dt_token: str) -> None:
...
         async with session.get(url) as resp:
             try:
                 return await resp.json()
-            except ConnectionError as err:
-                logging.exception("Error %s", err.message)  # pylint: disable=E1101
-                logging.warning(resp.status)
-                logging.warning(resp.text)
+            except aiohttp.ClientResponseError:
+                return {}
+            except aiohttp.ClientConnectionError:
```
[the remark this answers is not in the document]
Comment (kubealex) [reply extensions/eda/plugins/event_source/dt_esa_api.py:38]: Thx @Alex-Izquierdo, completely forgot to remove the returns I added to do the tests, will sanitize it and squash the commits

**Comment by kubealex** [comment]
It should be in a better shape now, squashed and cleaned up!

**Review by Alex-Izquierdo — APPROVED** [review]
LGTM

**Comment by agrimmer** [comment]
Thank you for the contribution!

**Pull request merged**